Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -215,6 +248,27 @@ public extension Sequence {
         }
     }
 
+    /// SwifterSwift: Returns a sorted sequence based on three map functions. Whenever the values of one map function match, the
+    /// next one will be used.
+    ///
+    /// - Parameters:
+    ///     - map1: Map function to sort by. Output type must be Comparable.
+    ///     - map2: Map function to sort by in case the values of `map1` match. Output type must be Comparable.
+    ///     - map3: Map function to sort by in case the values of `map1` and `map2` match. Output type must be Comparable.
+    func sorted<T: Comparable, U: Comparable, V: Comparable>(by map1: (Element) throws -> T,
+                                                             and map2: (Element) throws -> U,
+                                                             and map3: (Element) throws -> V) rethrows -> [Element] {
+        return try sorted {
+            if try map1($0) != map1($1) {
+                return try map1($0) < map1($1)
+            }
+            if try map2($0) != map2($1) {
+                return try map2($0) < map2($1)
+            }
```

Comment:
Same comment about repeated calculations.
```suggestion
            let value10 = map1($0)
            let value11 = map1($1)
            if try value10 != value11 {
                return try value10 < value11
            }
            
            let value20 = map2($0)
            let value21 = map2($1)
            if try value20 != value21 {
                return try value20 < value21
            }
```